magento/pwa-studio
# [PWA-836] Localize Cart

## Description

Translate strings within Cart

## Related Issue





https://jira.corp.magento.com/browse/PWA-836

## Acceptance 



### Verification Stakeholders

@dpatil-magento 
@davemacaulay 
@jimbo 
### Specification



### Verification Steps


1. Add a product to cart
2. Go to Cart page
3. Verify strings are translated where applicable
4. Remove all items from cart
5. Go to Cart page
6. Verify strings are translated where applicable

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
